Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/CalendarDatePickerSamplePage.xaml.cs

```diff
@@ -17,7 +17,9 @@
 
 namespace Uno.Gallery.Views.SamplePages
 {
+#if !__IOS__ && !__ANDROID__
```

Comment:
We should add macos there as well.

Reply:
Actually, I could just rely on the `xamarin:` namespace here, right?